application/models/jobfiles.model.php

```diff
@@ -10,49 +10,104 @@
 
 class JobFiles_Model extends CModel
 {
+
+    protected $dbVersionId;
+
+    protected function getDbVersionId(){
```

Comment:
This method should be part of Database_Model as it's specific to the catalog, and could be used by other components (not specific to JobFiles).